scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -77,6 +71,12 @@ workflow GvsCreateFilterSet {
     }
 
     # don't need to make these inputs because they should be created (based on inputs) and used only within this workflow
+    Int small_disk = select_first([small_disk_override, "100"])
+    Int medium_disk = select_first([medium_disk_override, "200"])
+    Int large_disk = select_first([large_disk_override, "300"])
+    Int huge_disk = select_first([huge_disk_override, "400"])
```

Comment:
should we set different values for these? maybe 200, 500, 1000, 2000

Reply:
done!